src/services/kpis/kpis.svc.coffee

```diff
@@ -260,7 +260,10 @@ angular
               (response) ->
                 # When no target has been defined
                 if response.data.error && response.data.error.code == 422
-                  # TODO force edit mode
+                  # TODO: this can be removed when Impac! API returns layout config for draft
+                  # mode KPIs. As then KPIs with no kpi.data and kpi.layout could be
+                  # considered in draft mode.
+                  kpi.is_draft = true
```

Comment:
Hmm was on automatic here, I think my brain was thinking I was mocking a back-end attribute. But you're right, this should not be snake_case.